buildpipeline/pipeline.json

```diff
@@ -7,632 +7,335 @@
   },
   "Pipelines": [
     {
-      "Name": "Trusted-All-Release-Linux",
+      "Name": "Trusted-All-Linux",
       "Parameters": {
         "TreatWarningsAsErrors": "false"
       },
-      "BuildParameters": {
-        "PB_ConfigurationGroup": "Release"
-      },
       "Definitions": [{
           "Name": "DotNet-CoreFx-Trusted-Linux",
           "Parameters": {
             "PB_DockerTag": "centos-7-d485f41-20173404063424",
-            "PB_BuildArguments": "-buildArch=x64 -Release -stripSymbols",
-            "PB_BuildTestsArguments": "-buildArch=x64 -Release -SkipTests -Outerloop -- /p:ArchiveTests=true /p:EnableDumpling=true",
+            "PB_BuildArguments": "-buildArch=x64 -$(PB_ConfigurationGroup) -stripSymbols",
```

Comment:
Are you intentionally removing a default value for PB_ConfigurationGroup here?  (may delete as I read more...)

Reply:
Yes I'm intentionally removing default values in the pipeline json file and instead defaulting it in the root pipebuild.